lib/state.rb

```diff
@@ -30,9 +34,9 @@ def self.from_s3 (s3_response)
   def self.extract_state_from_db_response (db_response)
     last_element = db_response.lazy.drop(db_response.count  - 1).first
     last_key = last_element["pcrKey"]
-    $logger.error('Missing pcrKey', db_response: db_response.to_a) if !last_key
-    # binding.pry if !last_key
-    { "cr_key" => last_key }
+    last_date_time = "'#{last_element["pcrDateTime"]}'"
```

Comment:
Why wrap this string in single quotes?

Reply:
I'm not sure if it is necessary but there were two reasons I thought it might be:
1. `last_date_time` gets passed into an SQL query. I haven't actually tested it without quotes but I've only seen DateTime comparisons with quotes
2. `last_date_time` can also be set in the env file, and spaces in environment variables can be finicky.

Reply:
It's true that you need to wrap dates in single quotes on mysql. If the value you're saving is `2020-04-18`, that'll need to somehow manifest in a sql query as `...WHERE pcrStartTime >= '2020-04-18'`. But it seems odd to include those single quotes in the bookmark file itself since they're not really the data but a sql formatting concern. This also means that in the S3 file, it will need to be represented like this:
```
  "last_date_time": "'2020-04-18'",
  "cr_key": 9876
```
And potentially in an env file as:
```
CR_KEY_START=9876
PCR_DATE_TIME_START="'2020-04-18'"
```
If anyone ever overlooks the single quote in one of those values when editing the env file or the S3 file, they'll produce a sql error. Seems better to represent the data in its simplest form and add quotes around the value at the point you're generating the sql